includes/abstracts/abstract-wc-product.php

```diff
@@ -1963,7 +1974,8 @@ public function get_file_download_path( $download_id ) {
 	public function get_price_suffix( $price = '', $qty = 1 ) {
 		$html = '';
 
-		if ( ( $suffix = get_option( 'woocommerce_price_display_suffix' ) ) && wc_tax_enabled() && 'taxable' === $this->get_tax_status() ) { // @phpcs:ignore Squiz.PHP.DisallowMultipleAssignments.Found, WordPress.CodeAnalysis.AssignmentInCondition.Found
+		// phpcs:ignore Squiz.PHP.DisallowMultipleAssignments.FoundInControlStructure, WordPress.CodeAnalysis.AssignmentInCondition.Found
```

Comment:
You can remove both lines and fix the conditional.

Reply:
Ok, thanks for spotting those. I'll wait in case someone else (or yourself) comes up with more, then fix all of them in a single refactoring commit.